src/DataStructures/Tensor/IndexType.hpp

```diff
@@ -39,16 +39,17 @@ namespace Frame {
 /// evaluate an analytic solution in that frame.
 struct FrameIsPhysical {};
 
-struct Logical {};
-struct Grid {};
-struct Inertial : FrameIsPhysical {};
+struct BlockLogical {};
+struct ElementLogical {};
+struct GlobalTimeIndependent {};
+struct Physical : FrameIsPhysical {};
```

Comment:
Please add a brief documentation what this frame means and why we chose this name; can be in a follow-up PR. We can add more detailed documentation about frames later.

As discussed in person, I'm not happy with the name `Physical` but don't have a better suggestion, so won't oppose this change. I'll keep trying to come up with a better name.

Reply:
Since there was the typo to fix I pushed a commit with the documentation anyway